src/test/java/LoginPage.java

```diff
@@ -0,0 +1,88 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.WebElement;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.titleIs;
+
+public class LoginPage extends PageBasis {
+    private static final String URL = "https://www.tut.by/";
+    private static final String LOGIN_TITLE = "Белорусский портал TUT.BY. Новости Беларуси и мира";
+    private static final By LOGIN_MENU_ITEM = By.xpath("//a[@class = 'enter']");
+    private static final By INPUT_LOGIN_USERNAME = By.xpath("//input[@name = 'login']");
+    private static final String LOGIN_USERNAME = "seleniumtests@tut.by";
+    private static final By INPUT_LOGIN_PASSWORD = By.xpath("//input[@name = 'password']");
+    private static final String LOGIN_PASSWORD = "123456789zxcvbn";
+    private static final By BUTTON_LOGIN_SUBMIT = By.xpath("//input[@class = 'button auth__enter']");
+    private static final By LABEL_LOGGED_USER_NAME = By.xpath("//span[@class = 'uname']");
+    private static final By LINK_LOGOUT_MENU_ITEM = By.xpath("//a[@class = 'enter logedin']");
+    private static final By BUTTON_LOGOUT = By.xpath("//a[@class = 'button wide auth__reg']");
+    private static final By LABEL_BUTTON_LOGOUT = By.xpath("//a[@class = 'enter']");
+
+    public LoginPage(WebDriver driver, WebDriverWait wait) {
+        super(driver, wait);
+    }
+
+    public void open() {
+        driver.get(URL);
+    }
+
+    public String loginTitle() {
```

Comment:
Move this variable in the class, in which you need it

Reply:
I removed this method and remained only the variable.